.github/ISSUE_TEMPLATE/cli_feature.yml

```diff
@@ -8,11 +8,27 @@ body:
   - type: textarea
     id: feature_description
     attributes:
-      label: Please describe your feature request
-      description: Please provide a detailed summary of the feature request including name of buildtest command (buildtest build) and options including intended behavior
+      label: Feature request description
```

Comment:
I kind of like this text to be a sentence instead of just be `Feature request description`.

It could just be  brief as `Please describe your feature` which is what we had.

This shows up as bold text above the text area. 
![Screen Shot 2023-03-24 at 1 58 01 PM](https://user-images.githubusercontent.com/12942230/227603870-fc698a5d-2b02-4a79-ad4a-ce869476f648.png)